sdk/Pulumi.Automation/PreviewOptions.cs

```diff
@@ -35,5 +35,10 @@ public sealed class PreviewOptions : UpdateOptions
         /// Refresh the state of the stack's resources before this preview.
         /// </summary>
         public bool? Refresh { get; set; }
+
+        /// <summary>
+        /// Show read logs in the preview.
```

Comment:
```suggestion
        /// Show resources that are being read in, alongside those being managed directly in the stack.
```

Lets just copy the cli help text for this.